addons/point_of_sale/static/src/xml/pos.xml

```diff
@@ -994,6 +994,10 @@
                 TOTAL
                 <span t-esc='widget.format_currency(receipt.total_with_tax)' class="pos-receipt-right-align"/>
             </div>
+            <div class="pos-receipt-amount">
```

Comment:
Your are not supposed to show ROUNDED and the value if you don't have rounding method set on the config. Put this block in a t-if